sdk-commons/src/main/java/org/openmrs/maven/plugins/model/DistroProperties.java

```diff
@@ -192,19 +193,16 @@ private List<Artifact> mergeArtifactLists(List<Artifact> childArtifacts, List<Ar
     }
 
     public void saveTo(File path) throws MojoExecutionException {
-        FileOutputStream out = null;
-        try {
-            out = new FileOutputStream(new File(path, DISTRO_FILE_NAME));
-            SortedProperties sortedProperties = new SortedProperties();
-            sortedProperties.putAll(properties);
-            sortedProperties.store(out, null);
-        }
-        catch (IOException e) {
+       // automatically close the stream after the try block finishes executing
+       try (FileOutputStream out = new FileOutputStream(new File(path, DISTRO_FILE_NAME))) { 
```

Comment:
A null check for the path argument could be added to avoid a NullPointerException at runtime